basil/HL/sensirion_sht85.py

```diff
@@ -6,17 +6,19 @@
 
 import numpy as np
 import logging
+import struct
 
 from basil.HL.SensirionBridgeDevice import SensirionBridgeI2CDevice
 
 logger = logging.getLogger(__name__)
+logging.getLogger("sensirion_shdlc_driver.connection").setLevel(logging.ERROR)
```

Comment:
this seems like a workaround for a problem in the driver? why is it necessary to do this here?

Reply:
When, data should be read but no data is received before the timeout, the sensirion_shdlc_driver writes a warning and throws a Exception. But in case of the asynchronous read we do not want to wait if no data is available and want to catch the Exception. I thought, in this case, we also do not want to print any warning. But maybe one can only ignore warings, if one is in the asynchronous mode?

Reply:
mhh, ok. a bit hacky but fine for me i guess..